pandas/core/generic.py

```diff
@@ -6596,9 +6596,10 @@ def _where(self, cond, other=np.nan, inplace=False, axis=None, level=None,
             return self._constructor(new_data).__finalize__(self)
 
     _shared_docs['where'] = ("""
-        Return an object of same shape as self and whose corresponding
-        entries are from self where `cond` is %(cond)s and otherwise are from
-        `other`.
+        Replace values that don't respect a `cond` condition.
```

Comment:
"don't respect" is too specific for `where` I think. Can you make this `` Replace values where the condition is %(cond)s  .  ``